spec/rest.spec.js

```diff
@@ -44,6 +44,66 @@ describe('rest create', () => {
       });
   });
 
+  it('should use objectId from client when allowCustomObjectId true', async () => {
+    config.allowCustomObjectId = true;
```

Comment:
Just wanted to say setting the config directly is a cool trick. Will definitely use this in future tests.